docs/conf.py

```diff
@@ -92,9 +92,10 @@ def __getattr__(cls, name):
 # built documents.
 #
 # The short X.Y version.
-version = '0.0'
+import fuel
+version = fuel.__version__[:fuel.__version__.rindex('.')]
```

Comment:
Sorry, I did try to find an exponential algorithm for that, but this is all I could do...